src/locales/uk.yaml

```diff
@@ -74,92 +78,141 @@ app:
       label:
         print_start_time: Відфільтрувати надруковане
         print_start_time_desc: Відфільтрувати вже надруковані деталі.
+        hidden_files: Відфільтрувати приховані файли
+        klipper_backup_files: Відфільтрувати резервні копії klipperа
     label:
-      dir_name: Ім'я каталогу
+      dir_name: Імʼя каталогу
       disk_usage: Використаний дисковий простір
+      diskinfo: Інформація про диск
       downloaded: Завантажене
-      file_name: Ім'я файлу
+      file_name: Імʼя файлу
       transfer_rate: Швидкість передачі даних
       uploaded: Завантажено
+      view_section_documentation: Перегляд '%{section}' документації
     msg:
       confirm: Ви впевнені? Це видалить всі файли і каталоги всередині.
       not_found: Файли не знайдені
       processing: Обробка
     overlay:
       label: <strong>Перетягніть</strong> файл сюди
+      drag_files_enqueue: <strong>Перетягніть</strong> файли сюди, щоб поставити їх у чергу
+      drag_files_folders_upload: <strong>Перетягніть</strong> файли та папки сюди, щоб завантажити
+      drag_files_folders: <strong>Перетягніть</strong> файли та папки сюди
     title:
       add_dir: Додати каталог
       add_file: Додати файл
+      command_palette: Команди
       download_file: Отримання файлу
       rename_dir: Перейменувати папку
       rename_file: Перейменувати файл
       upload_file: Завантаження файлу | Завантаження файлів
     tooltip:
       low_on_space: Мало місця на диску
+      root_disabled: Root недоступний. Перевірте файли логів.
+      url:
+        klipper_config: https://www.klipper3d.org/Config_Reference.html#%{hash}
+        moonraker_config: https://moonraker.readthedocs.io/en/latest/configuration/#%{hash}
+        moonraker_telegram_bot_config: >-
+          https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config#%{hash}
+    url:
+      klipper_config: https://www.klipper3d.org/Config_Reference.html
+      moonraker_telegram_bot_config: https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config
+      moonraker_config: https://moonraker.readthedocs.io/en/latest/configuration/
```

Comment:
This bit doesn't look correct, there should not be any `url` under `tooltip` as it currently shows

Reply:
I added them because in other languages this is done in such way (through a plugin for VS code i18n Ally)

Reply:
Thank you! Done.